bin/build.sh

```diff
@@ -19,7 +19,7 @@ rm package.json
 REPO="robertslando"
 IMAGE_NAME="zwave2mqtt"
 VERSIONS="$LATEST"
-TARGET_ARCHES="arm32v6 arm32v7 arm64v8 amd64"
+TARGET_ARCHES="arm32v6 arm32v7 arm64v8 arm64-v8 amd64"
```

Comment:
Are you sure that this is necessary? I think we just should just add the annotate flags that points to multiple variants `armv8` and `v8`

Reply:
No I am not sure, but I din't want to remove arm64v8 because it points to a annotate flag that maked arm64 and variant armv8
An alternative is to remove the variant armv8 and just replace it with v8, but I don't know the consequences for that.
At this point I just know it works

Reply:
This is how it will look
[my test version](https://hub.docker.com/r/magpern/zwave2mqtt/tags)